packages/blog/config/data/2020/2020-03-23-doctrine-entity-typed-properties-with-php74.md

```diff
@@ -207,7 +207,7 @@ What about return type? Look on the property type:
   * @return Collection|Training[]
   */
 -public function getTrainings(): iterable
-+public function getTrainings(): Collectoin
++public function getTrainings(): Collection
```

Comment:
The actual change is here.. I guess things are looking strange because of diff inside the blog post